Tests/SwiftFormatPrettyPrintTests/AssignmentExprTests.swift

```diff
@@ -30,53 +32,164 @@ public class AssignmentExprTests: PrettyPrintTestCase {
       """
       someVeryLongVariableName =
         anotherPrettyLongVariableName
-        && someOtherOperand
-      shortName = wxyz + aaaaaa
-        + bbbbbb + cccccc
-      longerName = wxyz + aaaaaa
-        + bbbbbb + cccccc || zzzzzzz
+          && someOtherOperand
+      shortName =
+        wxyz + aaaaaa + bbbbbb
+          + cccccc
+      longerName =
+        wxyz + aaaaaa + bbbbbb
+          + cccccc || zzzzzzz
 
       """
 
     assertPrettyPrintEqual(input: input, expected: expected, linelength: 30)
   }
 
-  public func testAssignmentFromFunctionCalls() {
+  public func testAssignmentOperatorFromSequenceWithFunctionCalls() {
+    let input =
+        """
+        result = firstOp + secondOp + someOpFetchingFunc(foo, bar: bar, baz: baz)
+        result = someOpFetchingFunc(foo, bar: bar, baz: baz)
+        result += someOpFetchingFunc(foo, bar: bar, baz: baz)
```

Comment:
For completeness, can you add a `+=` case that has a compound expression on the RHS?

Reply:
Done.